### PR TITLE
add endpoints, add dashboard

### PR DESCRIPTION
**Pull request by Northa**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: developers/staking.md [hunk 1]
```diff
@@ -26,6 +27,6 @@ Currently, the following staking interfaces exist for the
 
 Just connect your wallet to get started!
 
+- [https://alphab.ai/s/t/celestia-mocha-4/](https://alphab.ai/s/t/celestia-mocha-4/)
 - [https://explorer.kjnodes.com/celestia-testnet](https://explorer.kjnodes.com/celestia-testnet)
-- [https://test.anode.team/celestia](https://test.anode.team/celestia)
```
Comment (jcstein) [inline developers/staking.md:30]: hi @Northa why is this removed?

Comment (Northa) [reply developers/staking.md:30]: hi @jcstein have you tried to open that link? 
On my side it throws _Sorry, The chain is not found_  
Maintainer probably removed celestia testnet i.e link is no longer valid. 
So I removed it to keep a cleaner look

Comment (jcstein) [reply developers/staking.md:30]: thank you! good catch.

**Review by jcstein — APPROVED** [review]
